OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -14,15 +14,18 @@ struct AServer
 	var string IP;
 	var bool Locked;
 	var string GameMode;
+	var int iMaxPlayers,iNumPlayers;//1.5 - max players/current players
+	var string szGameType;//1.5 - localized game mode name
+	var string szMap;//1.5 - map name
+	var bool bWrongVersion;//1.5 - running same expansion mod - for future use? - needs to be inverted so only true value will trigger changing the menu
+	var int iPing;//1.5 - ping
 };
 var array<AServer> ServerList;//1.3 - not config - config is now in openserverlist, which handles loading the backup list
 var config string ServerURL;//0.8 server list URL to load
 var config string ServerListURL;//0.8 server list file to load
-
 var bool bServerSuccess;//0.8 got list of servers from online provider
-
-//1.5 ping update
-var OpenTimer Timer;
+var array<string> OpenQueries;//1.5 keep track of servers we've queried

```

Comment:
This array just holds IPs of current open queries.  To prevent multiple queries.  It gets cleared out if the user manually hits the refresh button.